robottelo/ui/base.py

```diff
@@ -47,41 +49,39 @@ def find_element(self, locator):
                 return _webelement
             else:
                 return None
-        except NoSuchElementException as e:
-            logging.debug("%s: Could not locate element %s.",
-                          type(e).__name__,
-                          locator[1])
-            return None
-        except Exception, error:
-            logging.debug("Failed to locate element. ERROR: %s", str(error))
+        except NoSuchElementException as err:
```

Comment:
@kbidarkar good insight, I've dropped the generic exception handling in order to see what else is being caught  by that, but will add the `TimeoutException`. Thanks